microsoft/testsuites/xdp/xdptools.py

```diff
@@ -88,6 +88,7 @@ def _install(self) -> bool:
         # install dependencies
 
         config_envs: Dict[str, str] = {}
+        arch = self.node.tools[Lscpu].get_architecture()
```

Comment:
can it be gotten from node.os.version? If not, it should be there for sharing, instead of loading from lscpu every time.

Reply:
it can be got from self.node.os.get_kernel_information().hardware_platform